envoy-control-tests/src/main/resources/envoy/config_ads_envoy1.yaml

```diff
@@ -0,0 +1,67 @@
+admin:
+  access_log_path: /dev/null
```

Comment:
Why we need additional envoy config files? Where they differ?

Reply:
Using one config for all test cases was error prone and confusing to me (e.g. domain dependencies refreshes polluted envoy logs) so I decided to create a separate small readable config just for one test. I'm not sure which approach is better but dumping every test case into one config becomes confusing after some time.